.github/workflows/base/ci.yml

```
@@ -0,0 +1,334 @@
+  security-scan:
+    name: Security Scan
+    runs-on: ubuntu-latest
+    needs: [lint-python, lint-typescript, lint-shell]
+    if: inputs.enable-security
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Run Snyk security scan
+        if: ${{ secrets.SNYK_TOKEN != null && secrets.SNYK_TOKEN != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)